sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/base64url.hpp

```diff
@@ -52,7 +52,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace _internal
           base64url.append("=");
           break;
         default:
-          throw new std::invalid_argument("Unexpected Base64URL encoding in the HTTP response.");
+          throw std::invalid_argument("Unexpected Base64URL encoding in the HTTP response.");
```

Comment:
How did you detect it here and do we have any other such instances?

Reply:
I was writing a comment on other review regarding throw and catch practices (https://github.com/Azure/azure-sdk-for-cpp/pull/2350#discussion_r640834624). And decided to check throws for another possible type of mistake. I searched for "throw new", this was the only place. We should not have any more usage in the code. To an extent, I myself did also develop a muscle memory to write "throw new" after years of C#, but when in C++, any `new` alerts me to check who owns it and who does the `delete`, plus with exceptions I know it is never a right thing to throw.

Reply:
> but when in C++, any `new` alerts me to check who owns it and who does the `delete`

That's a good thing we should keep an eye out for as part of regular PR review workflow. Nice.